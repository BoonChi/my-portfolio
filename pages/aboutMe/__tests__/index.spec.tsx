import { render, screen } from '@testing-library/react';
import AboutMe from '../index.page';

describe('About me', () => {
  it('renders About me page unchanged', () => {
    const { container } = render(<AboutMe />);
    expect(container).toMatchSnapshot();
  });

  it('renders and reads p text', () => {
    render(<AboutMe />);

    const heading = screen.getByText(/Poland/i);

    expect(heading).toBeInTheDocument();
  });
});
