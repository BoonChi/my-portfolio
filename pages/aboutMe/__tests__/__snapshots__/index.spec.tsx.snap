// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About me renders About me page unchanged 1`] = `
<div>
  <div
    class="main"
  >
    <div
      class="main-text"
    >
      <p>
        Currently I am based in 
        <strong>
          Poland
        </strong>
         and I
        <strong>
           DO NOT 
        </strong>
        require a visa sponsorship
      </p>
    </div>
    <div
      class="lists"
    >
      <ul>
        <strong>
          Frontend stack:
        </strong>
        <li>
          SASS
        </li>
        <li>
          React NextJs Framework
        </li>
        <li>
          Automation Unit test Jest
        </li>
        <li>
          Automation integration test Playwright
        </li>
      </ul>
      <ul>
        <strong>
          Backend stack:
        </strong>
        <li>
          Node.js(express)
        </li>
        <li>
          NoSql and Sql database
        </li>
        <li>
          Automation Unit test Jest
        </li>
      </ul>
      <ul>
        <strong>
          DevOps:
        </strong>
        <li>
          CI/CD(github)
        </li>
        <li>
          Docker
        </li>
      </ul>
    </div>
  </div>
</div>
`;
